packages/codemods/src/plugins/default.ts

```diff
@@ -14,6 +14,34 @@ const defaultCodemodPlugin: CodemodPlugin = {
         return newImport;
       },
 
+      buildAttributes({ originalNode, transformedNode, composedNode }) {
+        if (
+          originalNode.value.type === 'VariableDeclaration' ||
+          originalNode.value.type === 'TaggedTemplateExpression'
+        ) {
+          const newDeclaration = j(originalNode).replaceWith(transformedNode).get();
+
+          if (composedNode) {
+            let candidateForInsertion = newDeclaration;
+
+            while (
+              candidateForInsertion.parentPath &&
+              candidateForInsertion.parentPath.name !== 'body'
+            ) {
+              candidateForInsertion = candidateForInsertion.parentPath;
+            }
+
+            j(candidateForInsertion).insertBefore(composedNode);
+          }
+
+          return newDeclaration;
+        }
+
+        throw new Error(
```

Comment:
not sure if I understand this. You have the following condition:
```js
if (
  originalNode.value.type === 'VariableDeclaration' ||
  originalNode.value.type === 'TaggedTemplateExpression'
)
``` 
but here you mention `ExportNamedDeclaration` and `ExportDefaultDeclaration`

also, should we even `throw` at all or should we just skip?

Reply:
true, I can't think of any scenarios in the current implementation where it'll be something else. I'll change it to a warning